src/anm/ui/controls.js

```diff
@@ -235,6 +235,12 @@ Controls.prototype.react = function() {
     if (Controls.isInProgressArea(coords, w, h)) {
         if (coords.x > btnWidth && coords.x < w-btnWidth) {
             time = Math.round(p.state.duration*(coords.x-btnWidth)/(w-2*btnWidth));
+            if (time > p.anim.duration) {
+                //when the animation is something like 3.8 seconds long,
+                //the rounding will exceed the duration, which is not
+                //a good idea.
+                time = p.anim.duration;
```

Comment:
for whatever reason I thought `player.play()` would only take a whole number as a time value - i'll have this fixed, thanks!